Source/WebKit/WebKitSwift/RealityKit/RealityKitBridging.h

```diff
@@ -25,7 +25,7 @@
 
 #import <Foundation/Foundation.h>
 
-#if defined(TARGET_OS_VISION) && TARGET_OS_VISION
+#if ((defined(TARGET_OS_VISION) && TARGET_OS_VISION) || (defined(TARGET_OS_IOS) && TARGET_OS_IOS) || (defined(TARGET_OS_MAC) && TARGET_OS_MAC))
```

Comment:
I wonder if MODEL_ELEMENT (not MODEL_PROCESS) is a good flag to use here?

Reply:
or actually maybe MODEL_PROCESS is also a good proxy for this

Reply:
it seems `MODEL_PROCESS` is not defined in WebKitSwift :(